mono_repo/lib/src/presubmit_command.dart

```diff
@@ -115,8 +116,9 @@ Future<bool> presubmit(
       if (result.exitCode == 0) {
         stderr.writeln(green.wrap('(success)'));
       } else {
+        tmpDir ??= Directory.systemTemp.createTempSync();
```

Comment:
Put in a prefix w/ the package name – nice for cleanup if something fails...

Reply:
done